specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json

```diff
@@ -316,13 +314,7 @@
         ],
         "responses": {
```

Comment:
For this operation, linter is throwing an error

` A 'PATCH' operation 'NamedValue_Update' with x-ms-long-running-operation extension must have a valid terminal success status code 200 or 201.`

Reply:
Added "200" response

Reply:
The dummy 200 response represents the final response on the asynchronous operation. PATCH calls should be returning the full resource payload that is the result of the PATCH operation.

---
In reply to: [370430184](https://github.com/Azure/azure-rest-api-specs/pull/8225#discussion_r370430184) [](ancestors = 370430184)

Reply:
@majastrz, all our patch operations return 204 - no content. If we need to move them to 200 containing the payload, it would be a major change and can't be done in this iteration, but may be scheduled for the next version "2020-06-01". Could you please provide the link to "PATCH" requests requirements, so we could check other issues. Thank you.

Reply:
The synchronous PATCH contract is specified at https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource. The doc is vague and only says "and a response body". The body should generally be the full payload of the resource with the PATCH changes merged into it (basically the GET representation of what you saved in your storage in the end).

There's no specific doc for asynchronous PATCH, but general async stuff is documented at https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations. Generally when you make an operation async, what would normally be the synchronous response, becomes the final response on the final GET made to retrieve the status of the async operation.

As far as delaying this to next API version, I'll leave this for the current on-call for this week (@anthony-c-martin).

Reply:
@majastrz, I added payload to patch 200, now it returns the same as PUT (and  GET) operations